api/documents/DMStore.spec.ts

```diff
@@ -1,19 +1,17 @@
 import * as chai from 'chai'
-import {expect} from 'chai'
-import {Fields, File, Files} from 'formidable'
+import { expect } from 'chai'
+import { Fields, File, Files } from 'formidable'
 import 'mocha'
 import * as sinon from 'sinon'
 import * as sinonChai from 'sinon-chai'
-import {getConfigValue} from '../configuration'
-import {
-  SERVICES_DOCUMENTS_API_PATH,
-} from '../configuration/references'
+import { getConfigValue } from '../configuration'
+import { SERVICES_DOCUMENTS_API_PATH } from '../configuration/references'
+import { http } from '../lib/http'
+import { EnhancedRequest } from '../lib/models'
```

Comment:
do we still need EnhancedRequest as a type? you might just need the @types/express-session package